third_party/2and3/pynamodb/__init__.pyi

```diff
@@ -0,0 +1 @@
+__license__ = ...  # type: str
```

Comment:
Do you know if there's any tooling to do the conversion?  We've been using this internally for a while and just getting around to upstreaming it.